python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -908,43 +905,43 @@ def func(x):
 
     def get_program(self, dygraph_func, *args, **kwargs):
         """
-        Returns the translated static program and input/output variables from
+        Returns the translated static program and input/output Tensors from
         dygraph function. The users can use the program to run by executor.
 
         Args:
             dygraph_func (callable): the dygraph function.
-            *args, **kwargs : the input argument of dygraph_func.
+            *args (tuple): the input argument of dygraph_func.
+            **kwargs (dict): the input argument of dygraph_func.
 
         Returns:
             tuple of (main_program, startup_program, inputs, outputs) whose
-            types are (Program, Program, list of Variable, list of Variable).
+            types are (Program, Program, list of Tensors, list of Tensors).
             main_program: the converted main program.
             startup_program: the converted startup program.
-            inputs: list of input Variables which need to be fed.
-            outputs: list of output Variables which users can fetch.
+            inputs: list of input Tensors which need to be fed.
+            outputs: list of output Tensors which users can fetch.
 
         Examples:
             .. code-block:: python
 
-                import paddle.fluid as fluid
-                import numpy as np
+                import paddle
+
 
                 def func(x):
-                    x = fluid.dygraph.to_variable(x)
-                    if fluid.layers.mean(x) > 0:
+                    if paddle.mean(x) > 0:
                         x_v = x - 1
                     else:
                         x_v = x + 1
                     return x_v
 
-                prog_trans = fluid.dygraph.ProgramTranslator()
 
-                x = np.ones([1, 2])
+                prog_trans = paddle.jit.ProgramTranslator()
+                x = paddle.ones([1, 2])
                 main_prog, start_prog, inputs, outputs = prog_trans.get_program(func, x)
                 print([i.name for i in inputs])
-                # ['feed_0'] the feed input variable name representing x
+                # [u'generated_tensor_0'] the feed input tensor name representing x
```

Comment:
tensor -> Tensor
`Tensor` is a proper noun, better to capitalize its initial.

Reply:
Done